versions/3.0.md

```diff
@@ -345,14 +345,14 @@ Field Pattern | Type | Description
 #### <a name="pathsObject"></a>Paths Object
 
 Holds the relative paths to the individual endpoints.
-The path is appended to the [`basePath`](#oasBasePath) in order to construct the full URL.
-The Paths may be empty, due to [ACL constraints](#securityFiltering).
+The path is appended to the [`Server Object`](#serverObject) in order to construct
```

Comment:
Hmm, not sure how one can append a path (string) to an object. Maybe you want something like this?

> The path is appended to the base path build from the server object.